data/implementations/support/hyparquet.yaml

```diff
@@ -113,8 +113,8 @@ support:
     status: unknown
   format-data-page-v2:
     status: full
-  api-external-column-data:
-    status: full
+  api-parquet-summary-file:
+    status: none
```

Comment:
@platypii I looked at hyparquet documentation quickly and didn't see this mentioned but please let me know if I missed it.